integration/airflow/openlineage/airflow/listener.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import threading
+import uuid
+import attr
+
+from typing import TYPE_CHECKING, Optional, Callable
+
+from airflow.listeners import hookimpl
+
+from openlineage.airflow.adapter import OpenLineageAdapter
+from openlineage.airflow.extractors import ExtractorManager
+from openlineage.airflow.utils import DagUtils, get_task_location, get_job_name, get_custom_facets
+
+if TYPE_CHECKING:
+    from airflow.models import TaskInstance, BaseOperator
+    from sqlalchemy.orm import Session
+
+
+@attr.s(frozen=True)
+class RunData:
```

Comment:
`RunData` contains data specific to an active run, so I'd use `ActiveRun` instead

Reply:
👍